packages/material-ui/src/styles/createSpacing.js

```diff
@@ -55,7 +55,7 @@ export default function createSpacing(spacingInput = 8) {
           [
             'Material-UI: theme.spacing.unit usage has been deprecated.',
             'It will be removed in v5.',
-            'You can replace `theme.spacing.unit * 2` with `theme.spacing(2)`.',
+            'You can replace `theme.spacing.unit * y` with `theme.spacing(y)`.',
```

Comment:
I was migrating some old code to the new API, for one moment, I have tried a search and replace with `theme.spacing.unit * 2` while I was using `theme.spacing.unit * 3`. I want to remove the confusion.